From b9620f70a41e878b7f4e615ced4bd7ecf9fe7233 Mon Sep 17 00:00:00 2001
From: Albert Astals Cid <aacid@kde.org>
Date: Sun, 10 Nov 2024 23:47:18 +0100
Subject: [PATCH] Install ViewerPreferences again

I failed realizing it was usable given it's all inline functions
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bb1c73a05..a5c8b7610 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -720,6 +720,7 @@ if(ENABLE_UNSTABLE_API_ABI_HEADERS)
     poppler/UnicodeCClassTables.h
     poppler/UnicodeCompTables.h
     poppler/UnicodeDecompTables.h
+    poppler/ViewerPreferences.h
     poppler/XRef.h
     poppler/CharTypes.h
     poppler/ErrorCodes.h
-- 
GitLab

